app/controllers/admin/inbound_api_configurations_controller.rb

```diff
@@ -0,0 +1,46 @@
+###
+# Copyright 2016 - 2023 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module Admin
+  class InboundApiConfigurationsController < ::ApplicationController
```

Comment:
This controller needs a `before_action :require_ can_manage_inbound_api_configurations!`

Reply:
good catch. I'll add it.